assets/src/dashboard/animations/animationParts/bounce/index.js

```diff
@@ -69,7 +75,8 @@ const defaults = {
   duration: 1500,
 };
 
-export function WAAPIBounce(args) {
+export function AnimationBounce(args) {
+  const id = uuidv4();
```

Comment:
I'm using the `id` to link calls of `AMPAnimation` and `AMPAnimator` to each other.  They need to be separate because `AMPAnimator` has to appear as a direct child to an `amp-page`, but `AMPAnimation` wraps the actual element and it can be nested however deep it needs to be.